release.nix

```diff
@@ -22,29 +22,39 @@ let
       ${source}
     '';
   };
-in
-
-rec {
+  disciplina-wallet-macos-sandbox = writeShellScript ''
+    sandbox-exec -D HOME="$HOME" -D DYLD_ROOT_PATH="$DYLD_ROOT_PATH" \
+      -f ${./wallet/profile.sb} ${disciplina-wallet-wrapped}/bin/disciplina-wallet
+  '';
   disciplina-wallet = haskell.lib.justStaticExecutables project.disciplina-wallet;
 
   disciplina-wallet-wrapped = writeShellScript ''
     ${disciplina-wallet}/bin/dscp-wallet --witness https://witness.disciplina.io
   '';
 
-  disciplina-wallet-flatpak = buildFlatpak {
+disciplina-pkgs = rec {
+  inherit disciplina-wallet;
+
+  disciplina-wallet-flatpak = (buildFlatpak {
     app-id = "io.disciplina.Wallet";
     command = disciplina-wallet-wrapped;
     finish-args = [ "--share=network" ];
-  };
+  }).overrideAttrs (old: {
+    meta.platforms = lib.platforms.linux;
+  });
 
-  disciplina-wallet-macos-sandbox = writeShellScript ''
-    sandbox-exec -D HOME="$HOME" -D DYLD_ROOT_PATH="$DYLD_ROOT_PATH" \
-      -f ${./wallet/profile.sb} ${disciplina-wallet-wrapped}/bin/disciplina-wallet
-  '';
-
-  disciplina-wallet-macos-app = buildMacOSApp {
+  disciplina-wallet-macos-app = (buildMacOSApp {
     name = "Disciplina";
     target = disciplina-wallet-macos-sandbox;
     withOpen = true;
-  };
+  }).overrideAttrs (old: {
+    meta.platforms = lib.platforms.darwin;
+  });
+};
+in
+disciplina-pkgs // {
+  all-buildable = with lib; let
+      system = lib.systems.elaborate { inherit (pkgs) system; };
+      match = lib.meta.platformMatch system;
+    in filterAttrs (name: pkg: any match pkg.meta.platforms) disciplina-pkgs;
```

Comment:
I'm not sure there's a point in making `release.nix` fully buildable, that's why I moved this stuff to `release.nix` in the first place.

Reply:
I want to be able to do `nix build -f ./release.nix all-buildable`, that's why